### PR TITLE
Update Electron Dependency and Fix Linux Icons

### PR DESCRIPTION
**Pull request by droghio**
This fixes a few UI bugs including the lack of an icon under Linux ie #371.

### CODE REVIEW

**Comment by avahowell** [comment]
Can you manually verify that electron successfully launches when you run `npm start`?

**Comment by droghio** [comment]
I can do that and one better. Works with npm start, and I also made releases for Mac, Linux, and Windows and made sure they all launch.

**Comment by avahowell** [comment]
perfect, just didn't want to have a similar 'test-pass-but-electron-is-broken' scenario as the last time we updated electron.

LGTM

File: package.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
     "sia-import": "(test -d Sia || mkdir -p Sia) && (cp $GOPATH/bin/siad Sia & cp $GOPATH/bin/siac Sia)",
     "sia-kill": "killall siad; while pgrep siad; do sleep .5; done;",
     "sia-refresh": "npm run sia-kill && rm -rf Sia && npm run sia-import",
-    "release-args": "webpack && electron-packager . Sia-UI --app-version=1.0.1 --version=1.2.2 --overwrite --prune --ignore='(node_modules|release|config.json|errors.log)' --out=release/v1.0.1 --icon=assets/icon",
+    "release-args": "webpack && electron-packager . Sia-UI --app-version=1.0.2 --overwrite --prune --ignore='(node_modules|release|config.json|errors.log)' --out=release/v1.0.2 --icon=assets/icon",
```
Comment (lukechampine) [inline package.json:61]: what is the effect of removing the `--version` arg? Will it just use the most recent version?

**Comment by droghio** [comment]
Not exactly, from the electron-packager docs:

> The Electron version with which the app is built (without the leading 'v') - for example, 0.33.9. See 
> Electron releases for valid versions. If omitted, it will use the version of the nearest local installation 
> of electron-prebuilt.

It will default the version of electron prebuilt installed on the system, which is currently defined as `^1.2.8` == `>=1.2.8 < 2.0.0`.

**Pull request merged**